include/tester/db/Database.hpp

```diff
@@ -138,29 +60,33 @@ class Database
         return result.extract();
     }
 
-    std::vector<bsoncxx::document::value> get_all_documents(const std::string& collection_name)
+    std::vector<bsoncxx::document::value> getAllDocuments(std::string collectionName)
     {
-        mongocxx::collection collection = _database[collection_name];
+        mongocxx::collection collection = _database[collectionName];
         mongocxx::cursor cursor = collection.find({});
 
         std::vector<bsoncxx::document::value> results;
-        for (auto&& doc : cursor) {
+        for (auto&& doc : cursor) 
+        {
             results.emplace_back(std::move(doc));
         }
 
-        return results;
+        return std::move(results);
     }
 
-    bsoncxx::stdx::optional<std::string> get_field(const std::string& collection_name, const std::string& id, const std::string& field_name)
+    bsoncxx::stdx::optional<std::string> getField(std::string collectionName, std::string id, std::string field_name)
```

Comment:
same here regarding the `static constexpr`

Reply:
done